### PR TITLE
Feature/fixes errors for jenkins

### PR DESCRIPTION
**Pull request by jcalderin**

### CODE REVIEW

File: test/acceptance/integration/tools/general_utils.py [hunk 1]
```diff
@@ -128,7 +127,16 @@ def kill(proc_pid):
 
 def drop_database(ip, port, database):
```
Comment (iariasleon) [inline test/acceptance/integration/tools/general_utils.py]: Insert documentation string stub

File: test/acceptance/integration/tools/general_utils.py [hunk 2]
```diff
@@ -128,7 +127,16 @@ def kill(proc_pid):
...
     if database == "":
-        pymongo.Connection(ip, port).drop_database('orion')
+        pymongo.Connection(ip, port).drop_database('acceptance')
     else:
-        pymongo.Connection(ip, port).drop_database('orion-{service}'.format(service=database))
+        pymongo.Connection(ip, port).drop_database('acceptance-{service}'.format(service=database))
+
+
+def drop_all_test_databases(ip, port):
```
Comment (iariasleon) [inline test/acceptance/integration/tools/general_utils.py]: Insert documentation string stub

Comment (jcalderin) [inline test/acceptance/integration/tools/general_utils.py]: fixed in 4bc1aa8

File: test/acceptance/integration/tools/general_utils.py (hunk 1 above)
Comment (jcalderin) [inline test/acceptance/integration/tools/general_utils.py]: Fixed in 4bc1aa8

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**